### PR TITLE
Account data

### PR DESCRIPTION
**Pull request by mfairchild365**
This PR adds support for retrieving account data, such as the correct email address and primary affiliation. It tries to get the data from LDAP first, and falls back to directory data. See the readme for more info.

https://github.com/unlcms/UNL-CMS-2/issues/3

Note, we still need to implement an account search and import feature. But everything is set up now to make that pretty easy (crosses fingers)

### CODE REVIEW

File: src/Helper.php [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      $user->setUsername($username);
+      $user->setEmail($username . '@unl.edu');
+      $user->enforceIsNew();
+      $user->activate();
+    }
+    
+    $user->save();
+    
+    //The first time that they log in, try to update userdata
+    $this->updateUserData($user);
```
Comment (ericras) [inline src/Helper.php]: This is going to run every time a user logs in.  If the idea is to only have this run on new account creation, and let unl_cas_user_login() handle adding to the queue on subsequent logins, shouldn't this be up inside the "if (!$user) {}" block on line 28?

Comment (mfairchild365) [reply src/Helper.php]: Hah, probably. Good catch.

Comment (ericras) [reply src/Helper.php]: importUser() probably isn't the best name for the method.  initializeUser?

Comment (mfairchild365) [reply src/Helper.php]: I was just copying it...

Comment (ericras) [reply src/Helper.php]: I realize that.  Just saying that the name was probably always bad.

**Comment by kabel** [comment]
Broke the build :(

eb3ac77 change a method that is used in line 76 of the Controller.

**Pull request merged**